docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -78,17 +75,74 @@ The example above uses the `env` variable, which corresponds to the value of the
 
 [discrete]
 [[synthetics-cli-params]]
-= Use CLI arguments to set params
+== CLI argument
 
 To set parameters when running `elastic-synthetics` on the command line, use the `--params` or `-p` flag to the `elastic-synthetics` program. The provided map is merged over any existing variables defined in the `synthetics.config.{js,ts}` file.
 
-For example, to override the `url` parameter, you would run:
+For example, to override the `my_url` parameter, you would run:
 
 [source,sh]
 ----
-elastic-synthetics . --params '{"url": "http://localhost:8080"}'
+elastic-synthetics . --params '{"my_url": "http://localhost:8080"}'
 ----
 
+[discrete]
+[[synthetics-params-secrets-use]]
+= Use params
+
+You can use params in both lightweight and browser monitors created in
+either a project or the Synthetics UI in {kib}.
+
+[discrete]
+[[synthetics-params-secrets-use-project]]
+== In a project
+
+For lightweight monitors in projects, wrap the name of the param in `${}` (for example, `${my_url}`).
+
+[source,yaml]
+----
+- type: http
+  name: Todos Lightweight
+  id: todos-lightweight
+  urls: ["${my_url}"]
+  schedule: '@every 1m'
+----
+
+In browser monitors, parameters can be referenced via the `params` property available within the 
+argument to a `journey`, `before`, `beforeAll`, `after`, or `afterAll` callback function.
+
+Add `params.` before the name of the param (for example, `params.my_url`):
+
+[source,js]
+----
+beforeAll(({params}) => {
+  console.log(`Visiting ${params.my_url}`)
+})
+
+journey("My Journey", ({ page, params }) => {
+  step('launch app', async () => {
+    await page.goto(params.my_url) <1>
+  })
+})
+----
+<1> If you are using TypeScript, replace `params.my_url` with `params.my_url as string`.
+
+[discrete]
+[[synthetics-params-secrets-use-ui]]
+== In the UI
+
+To use a param in a lightweight monitor that is created in the Synthetics UI,
+wrap the name of the param in `${}` (for example, `${my_url}`).
+
+[role="screenshot"]
+image::images/synthetics-params-secrets-kibana-use-lightweight.png[Use a param in a lightweight monitor created in the Synthetics UI]
+
+To use a param in a browser monitor that is created in the Synthetics UI,
+add `params.` before the name of the param (for example, `params.my_url`).
+
+[role="screenshot"]
+image::images/synthetics-params-secrets-kibana-use-browser.png[Use a param in a browser monitor created in the Synthetics UI]
+
 [discrete]
 [[synthetics-secrets-sensitive]]
 = Working with secrets and sensitive values
```

Comment:
I'm not sure what to do with [this section](https://observability-docs_2922.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-params-secrets.html#synthetics-secrets-sensitive).

Reply:
@dominiqueclarke and/or @shahzad31 can you take a look at [this section](https://observability-docs_2922.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-params-secrets.html#synthetics-secrets-sensitive) and let me know if there's anything we should change?